src/split-panel/index.tsx

```diff
@@ -74,15 +74,15 @@ const TransitionContentSide = ({
       className={clsx(baseProps.className, styles.drawer, styles.root, styles['position-side'], {
         [styles['drawer-closed']]: !isOpen,
       })}
+      style={{
+        top: topOffset,
+        bottom: bottomOffset,
+        width: isOpen ? cappedSize : undefined,
+        maxWidth: isRefresh ? '100%' : undefined,
```

Comment:
This additional styling breaks the appearance of the split panel. I reverted the merged PR (#227). Could you elaborate what the additional styling is intended to do? Also, please verify the changes in your browser on the classic theme.

Steps to reproduce (tested on Chrome, Safari, FF, Classic theme):
- Open the app-layout/with-split-panel example page in classic.
- You'll see the additional spacing at the top of the split panel.